source/bricklaunch

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+#
+# bricklaunch.sh <ops> <program>
+# - open a new vt and switch to it
+# - give 'user' ownership of the vt
+# - run the given programm on the new vt as 'user'
+#
+
+OPTIND=1
+user='user'
+openvt_opts=""
+rm_ttyfile=0
+
+function show_help {
+  cat <<END_OF_HELP
+bricklaunch.sh <opts> <program>
+
+  -h        this help
+  -s        switch to the new virtual terminal
```

Comment:
I definitely need this to be an option. For openroberta I will launch the connector service at boot-time (if installed and enabled). The service will be started via bricklaunch to not run as root, but have a new vt. Still I don't want the vt to be shown yet. I'll try if I can use systemd feature in the unit file to achieve the same.